src/index.js

```diff
@@ -1,122 +1,142 @@
-//         filteredList[i].index = i +1
-//     }
-//     saveTask(filteredList)
-// }
-// ----------------------end storage-----------------------
 
 
 function displayitems(chores){
     const div = document.createElement("div");
 div.innerHTML = `
 <div class ="checkclass">
-<input class= "checklist" type="checkbox" id="inputId${chores.index}">
-<input type="text" class="class-input" value="${chores.description}" readonly>
+<button class= "checkbox" id="check${chores.index}"></button>
+<input type="text" id="${chores.index}" class="class-input inputBox${chores.index}" value="${chores.description}" readonly>
 </div>
 <div>
-<button class="edit">edit</button><button class="del-btn"> del</button> <button class="fa-solid fa-ellipsis-vertical">del</button> </div>
+<button class="del-btn del-btn-${chores.index} del-display" id="trash-${chores.index}"><i  class="delIcon fa-solid fa-trash-can"></i></button>
+<span class="edit-btn editbtn${chores.index} "><i id="edit-id" class=" trashIcon fa-solid fa-ellipsis-vertical"></i></span>
+ </div>
 
 
 `;
-
 div.classList.add("listItems");
-cont.appendChild(div);
- 
-div.addEventListener('click', function(e){
-if(e.target.classList.contains('edit')){
-    console.log('it contains');
-    // e.target.parentNode.parentNode.children[1].removeAttribute('readonly');
-    let inputEdit = e.target.parentNode.parentNode.children[0].children[1];
-    inputEdit.toggleAttribute('readonly') ;
-    if (!inputEdit.hasAttribute('readonly')){
-        e.target.innerHTML = "save";
+cont.appendChild(div); 
+// -------------------------table buttons-------------------------
+ const editBtn = document.querySelector(`.editbtn${chores.index}`);
+ const divInput = document.querySelector(`.inputBox${chores.index}`)
+ const trashCan = document.querySelector(`.del-btn-${chores.index}`);
+
+
+
+
+const checkbox = document.querySelector(`#check${chores.index}`);
+if(checkbox.textContent === '\u2714'){
+    checkbox.nextElementSibling.classList.add('crossinput');
+}
+checkbox.addEventListener('click', function(e){
+//    const btnindex = e.target.id.slice(5);
+   editBtn.classList.toggle('edit-display');
+   trashCan.classList.toggle('del-display');
+
+   if(e.target.textContent === '\u2714'){
+       
+       e.target.textContent= ' ';
+       divInput.classList.remove('crossinput');
+       chores.completed = false;
+       const localData = JSON.parse(localStorage.getItem('listItem'));
+       localData[chores.index -1].completed = false;
+       localStorage.setItem('listItem', JSON.stringify(localData));
+     
+       console.log(chores.completed)
+       
+   }else{
+    e.target.textContent= '\u2714';
+    divInput.classList.add('crossinput');
+    chores.completed = true;
+    const localData = JSON.parse(localStorage.getItem('listItem'));
+    localData[chores.index - 1].completed = true;
+    localStorage.setItem('listItem', JSON.stringify(localData))
+
+    console.log(chores.completed)
+   }
+})
```

Comment:
- [ ] According to project requirements, the function for updating the status of the `completed` key for tasks when the checkbox is checked should be in another file in your modules directory. You can then import it into your `index.js` file. Kindly implement it. 👍